monai/transforms/lazy/utils.py

```diff
@@ -129,6 +129,6 @@ def resample(data: torch.Tensor, matrix: NdarrayOrTensor, spatial_size, kwargs:
         "padding_mode": kwargs.pop(LazyAttr.PADDING_MODE, None),
     }
     resampler = monai.transforms.SpatialResample(**init_kwargs)
-    # resampler.lazy_evaluation = False
-    with resampler.trace_transform(False):  # don't track this transform in `data`
+    # resampler.lazy_evaluation = False  # resampler is a lazytransform
```

Comment:
This line need to keep in comment since `lazy_evaluation` is not implemented in this PR